Vic2ToHoI4/Source/HOI4World/Ideologies/Ideology.cpp

```diff
@@ -16,7 +16,7 @@ HoI4::Ideology::Ideology(const std::string& _ideologyName, std::istream& theStre
 		dynamicFactionNames = namesStrings.getStrings();
 	});
 	registerKeyword("color", [this](const std::string& unused, std::istream& theStream) {
-		theColor = std::make_unique<commonItems::Color>(commonItems::Color::Factory::getColor(theStream));
+		theColor = std::make_unique<commonItems::Color>(commonItems::Color::Factory{}.getColor(theStream));
```

Comment:
Is this correct?

Reply:
Should be. We'll make sure the build is happy.